src/puppetlabs/trapperkeeper/authorization/ring.clj

```diff
@@ -16,16 +16,16 @@
 
 ;; Functions
 
-(schema/defn authorized-authentic? :- schema/Bool
+(schema/defn authorized-authenticated :- schema/Bool
   "Get whether the authorized client is considered authentic or not."
   [request :- Request]
-  (get-in request [:authorization :authentic?]))
+  (get-in request [:authorization :authenticated]))
 
 (schema/defn set-authorized-authentic? :- Request
```

Comment:
And since we're in here again might as well update the docstrings on these two functions with s/authentic/authenticated